trade-finance-manager-ui/component-tests/assertions.ts

```diff
@@ -27,7 +27,7 @@ export const assertions = (wrapper: CheerioAPI, html: string, params: object) =>
       expect(wrapper(selector).attr('href')).toBeUndefined();
       expect(wrapper(selector).attr('disabled')).toEqual('disabled');
     },
-    toLinkTo: (href: string, text: string) => {
+    toLinkTo: (href: string | undefined, text: string) => {
```

Comment:
It shouldn't really - this is coming from the fact that we are creating buttons which don't link anywhere. That being said, it is probably better to set the button href to `'#'` so that we don't have to make this change to the type here